peering/models/models.py

```diff
@@ -524,6 +525,8 @@ class DirectPeeringSession(BGPSession):
     router = models.ForeignKey(
         "Router", blank=True, null=True, on_delete=models.SET_NULL
     )
+    interface = models.CharField(max_length=200, blank=True)
```

Comment:
The only other thing that I would point out is to ensure the documentation is updated for the DirectPeeringSession model here https://github.com/peering-manager/peering-manager/blob/435f75e87cbb49941e10c7bd5a4663ae398867bc/docs/models/peering/directpeeringsession.md

Sorry, forgot to include this in the previous review! :)